src/devices/onokom.ts

```diff
@@ -0,0 +1,2093 @@
+import {Zcl} from "zigbee-herdsman";
+import * as m from "../lib/modernExtend";
+import * as reporting from "../lib/reporting";
+import type {DefinitionWithExtend} from "../lib/types";
+
+const manufacturerCode = 0x4703;
+
+export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["AUX-1-ZB-S"],
+        model: "AUX-1-ZB-S",
+        vendor: "ONOKOM",
+        description: "ONOKOM AIR AUX-1-MB-B (Adapter for household and semi-industrial AUX systems)",
+        ota: true,
+        extend: [
+            m.enumLookup({
+                name: "AC connected",
+                cluster: "hvacThermostat",
+                attribute: {ID: 0x4734, type: Zcl.DataType.ENUM8},
+                lookup: {
+                    Disconnected: 0,
+                    "Invalid data recieved": 1,
+                    "Connected with issues": 2,
```

Comment:
```suggestion
                    "connected_with_issues": 2,
```

These should also be `snake_case`

Reply:
@Ziracul I don't think this has been solved?

Reply:
@Koenkk Updated, my apologies!